docs/setup/upgrade/saved-objects-migration.asciidoc

```diff
@@ -3,7 +3,7 @@
 
 Each time you upgrade {kib}, an upgrade migration is performed to ensure that all <<managing-saved-objects,saved objects>> are compatible with the new version.
 
-NOTE: To help you prepare for the upgrade to 8.0.0, 7.17.0 includes an https://www.elastic.co/guide/en/kibana/7.17/upgrade-assistant.html[*Upgrade Assistant*].
+NOTE: {kib} includes an {kibana-ref-all}/7.17/upgrade-assistant.html[*Upgrade Assistant*] to help you prepare for an upgrade.
```

Comment:
Likewise, if you want to link to the revived upgrade-assistant page, rather than the 7.17 page, you can do this now:
```suggestion
NOTE: {kib} includes an <<upgrade-assistant,*Upgrade Assistant*>> to help you prepare for an upgrade.
```